Public/Src/Utilities/UnitTests/Executables/TestProcess/Operation.cs

```diff
@@ -473,11 +474,15 @@ public static Operation CreateDir(FileOrDirectoryArtifact path, bool doNotInfer
         /// Creates a write file operation that appends. The file is created if it does not exist.
         /// Writes random content to file at path if no content is specified.
         /// </summary>
-        public static Operation WriteFile(FileArtifact path, string content = null, bool doNotInfer = false, bool changePathToAllUpperCase = false)
+        public static Operation WriteFile(FileArtifact path, string content = null, bool doNotInfer = false, bool changePathToAllUpperCase = false, bool useLongPathPrefix = false)
         {
+            Contract.Assert(!changePathToAllUpperCase || !useLongPathPrefix, "Cannot specify changePathToAllUpperCase and useLongPathPrefix simultaneously");
+
+            string additionalArgs = changePathToAllUpperCase ? Operation.AllUppercasePath : useLongPathPrefix ? Operation.UseLongPathPrefix : null;

```

Comment:
would you mind adding parenthesis (or improve multi-line) to indicate if the ? operator is left or right assiciative?